ui-admin/src/portal/siteContent/SiteContentEditor.tsx

```diff
@@ -251,30 +259,68 @@ const SiteContentEditor = (props: InitializedSiteContentViewProps) => {
-                }
-                if (hasInvalidSection) {
-                  return 'Site is invalid. Correct to save.'
-                }
-                return 'Save changes'
-              })()}
-              onClick={() => createNewVersion(workingContent)}>
+        <div className="d-flex flex-grow-1 justify-content-end">
+          {
+            isEditable && <>
+              <Button className="me-md-2" variant="primary"
+                disabled={readOnly || hasInvalidSection || (siteContent === workingContent)}
+                tooltipPlacement={'left'}
+                tooltip={(() => {
+                  if (siteContent === workingContent) {
+                    return 'Site is unchanged. Make changes to save.'
+                  }
+                  if (hasInvalidSection) {
+                    return 'Site is invalid. Correct to save.'
+                  }
+                  return 'Save changes'
+                })()}
+                onClick={() => createNewVersion(workingContent)}>
                   Save
-            </Button>
-            {
-            // eslint-disable-next-line
+              </Button>
+              {
+                // eslint-disable-next-line
             // @ts-ignore  Link to type also supports numbers for back operations
-              <Link className="btn btn-cancel" to={-1}>Cancel</Link>
-            }
+                <Link className="btn btn-cancel" to={-1}>Cancel</Link>
+              }
+            </>
+          }
+          <a className="btn" href="#" role="button" data-bs-toggle="dropdown" aria-expanded="false">
+            Actions <FontAwesomeIcon className="ms-2" icon={faEllipsisV}/>
+          </a>
+          <div className="dropdown-menu">
+            <ul className="list-unstyled">
+              <li onClick={() => setShowBrandingModal(true)}>
+                <a className="dropdown-item" onClick={() => setShowBrandingModal(true)}>
+                  Branding <FontAwesomeIcon icon={faPalette} className="fa-lg"/>
+                </a>
+              </li>
+              <li>
+                <Link to="../media" className="dropdown-item">
+                  Manage media <FontAwesomeIcon icon={faImage} className="fa-lg"/>
+                </Link>
+              </li>
+              { portalEnv.preRegSurveyId &&
+                <li>
+                  <Link to={'../forms/preReg'}>
+                    Pre-registration <FontAwesomeIcon icon={faClipboard} className="fa-lg"/>
+                  </Link>
+                </li> }
+              { !portalEnv.preRegSurveyId &&
+                <li>
+                  <a className="dropdown-item" onClick={() => setShowAddPreRegModal(true)}>
+                    Pre-registration <FontAwesomeIcon icon={faClipboard} className="fa-lg"/>
+                  </a>
+
+                </li>
+              }
+              <li >
+                <a className="dropdown-item" onClick={() => setShowTranslationModal(true)}>
+                  Translations
```

Comment:
Since all the other items in the list have a fontawesome icon maybe it could be cool to do:
```suggestion
                  Translations <FontAwesomeIcon icon={faGlobe} className="fa-lg"/>
```